man/man7/zpoolprops.7

```diff
@@ -28,7 +28,7 @@
 .\" Copyright (c) 2021, Colm Buckley <colm@tuatha.org>
 .\" Copyright (c) 2023, Klara Inc.
 .\"
-.Dd July 29, 2024
+.Dd June 20, 2024
```

Comment:
nit: no need to go backwards in time!  Let's either drop the date change or update it to today.